lib/kennel/parts_serializer.rb

```diff
@@ -8,13 +8,9 @@ def initialize(filter:)
 
     def write(parts)
       Progress.progress "Storing" do
-        if filter.tracking_id_filter
-          write_changed(parts)
-        else
-          old = old_paths
-          used = write_changed(parts)
-          (old - used).uniq.each { |p| FileUtils.rm_rf(p) }
-        end
+        existing = existing_paths
+        used = write_changed(parts)
+        FileUtils.rm_rf((existing - used).uniq)
```

Comment:
... the .uniq should not be needed

Reply:
In what sense? That the data can never contain duplicates, or that it's OK to allow duplicates through?

Reply:
existing could never have duplicates since it's just a list of files on disk

Reply:
Right (but only because we `uniq` the project / tracking_id filters); and `used` can, but that doesn't matter because that's the removal list. Got it 👍